src/placeos-driver/proxy/remote_driver.cr

```diff
@@ -231,12 +213,11 @@ module PlaceOS::Driver::Proxy
     def debug
       module_id = module_id?
       raise Error.new(ErrorCode::ModuleNotFound, "could not find module id", *@error_details) unless module_id
-
-      core_uri = which_core(module_id)
-
-      # build request
-      core_uri.path = "/api/core/v1/command/#{module_id}/debugger"
-      HTTP::WebSocket.new(core_uri)
+      Core::Client.client(which_core, request_id) do |client|
+        client.debug(module_id) do |message|
+          yield message
+        end
+      end
```

Comment:
This looks to actually be breaking change. it used to return a websocket. 
We may need to add a method to the client that returns a websocket with the messages, but tbh this is a better way of going about it.